src/features/auth/projectRedactorSignup/RedactorSignupConfirmationEmailSent.tsx

```diff
@@ -0,0 +1,74 @@
+import { t } from '@lingui/macro'
+import { useNavigation } from '@react-navigation/native'
+import { StackScreenProps } from '@react-navigation/stack'
+import React, { FunctionComponent } from 'react'
+import { openInbox } from 'react-native-email-link'
+
+import {
+  CenteredText,
+  Description,
+  EmailSentModalContent,
+} from 'features/auth/components/signupComponents'
+import { contactSupport } from 'features/auth/support.services'
+import { navigateToHome } from 'features/navigation/helpers'
+import { RootStackParamList, UseNavigationType } from 'features/navigation/RootNavigator'
+import { analytics } from 'libs/analytics'
+import { BottomContentPage } from 'ui/components/BottomContentPage'
+import { ButtonPrimary } from 'ui/components/buttons/ButtonPrimary'
+import { ButtonTertiary } from 'ui/components/buttons/ButtonTertiary'
+import { ModalHeader } from 'ui/components/modals/ModalHeader'
+import { ArrowPrevious } from 'ui/svg/icons/ArrowPrevious'
+import { Close } from 'ui/svg/icons/Close'
+import { Email } from 'ui/svg/icons/Email'
+import { ExternalSite } from 'ui/svg/icons/ExternalSite'
+import { Spacer, Typo } from 'ui/theme'
+
+type Props = StackScreenProps<RootStackParamList, 'RedactorSignupConfirmationEmailSent'>
+
+export const RedactorSignupConfirmationEmailSent: FunctionComponent<Props> = ({ route }) => {
+  const { goBack } = useNavigation<UseNavigationType>()
+
+  function onClose() {
+    navigateToHome()
+  }
+
+  function onConsultHelpSupport() {
+    analytics.logHelpCenterContactSignupConfirmationEmailSent()
+    contactSupport.forSignupConfirmationEmailNotReceived()
+  }
+
+  return (
+    <BottomContentPage>
+      <ModalHeader
+        title={t`Confirmez votre e\u2011mail`}
+        leftIcon={ArrowPrevious}
+        onLeftIconPress={goBack}
+        rightIcon={Close}
+        onRightIconPress={onClose}
```

Comment:
```suggestion
        onRightIconPress={navigateToHome}
```

=> plus lisible et très légèrement mieux en terme de perf